docs/develop/nova/block_explorer.md

```diff
@@ -1,18 +1,17 @@
----
-title: Subspace block explorer
-sidebar_position: 10
-description: Subspace hosted block explorer
-keywords:
-  - block explorer
-  - subspace network
----
+# Subspace block explorer
 
 ### Subspace Block Explorer
 
-**[Block explorer link](https://explorer.subspace.network/#/gemini-3g)**
-
 This early version provides a clear and user-friendly visualization of Subspace-specific statistics that cater to the needs of our farmers and developers.
 
-On the top of the page, you can easily toggle between **all available networks** and **EVM**. 
+On the top of the page, you can easily toggle between *all available networks and EVM.*
+
+The following block explorers provide information on Subspace .
+
+
+
+| Block Explorer   name | URL |
+|------------------------------|----------------------------------------|
+| Astral  | [explorer.subspace.network](https://explorer.subspace.network/#/gemini-3g/consensus)   |
+| Subscan  | [subspace.subscan](https://subspace.subscan.io/) |
```

Comment:
I suggest leaving the image of Astral (Subspace Explorer) right after the table, as it's a preferred tool to view the data. 

![BlockExplorer-1](/img/developers/BlockExplorer-1.png)

Reply:
should i also add subscan image also

Reply:
Sure, let's add one for Subscan as well. I guess in this case, we can omit to put the resources in a table, and just put them one after another with one screenshot for each. Thanks!

Reply:
okkaay